frontend/components/HeaderButton.js

```diff
@@ -6,15 +6,10 @@ import IconButton from './IconButton';
 */
 
 export default function HeaderButton(props) {
-    const {
-        onPress,
-        name,
-    } = props;
     return (
         <IconButton
+            {...props}
             size={20}
-            onPress={onPress}
-            name={name}
             containerStyle={{ paddingLeft: 7, paddingRight: 7 }}
         />
     )
```

Comment:
Does this work cross-platform? I know we had issues with that with the previous version.